app/models/placement_need.rb

```diff
@@ -28,6 +28,6 @@ def date_in_future
   end
 
   def sanitize_input
-    self.postcode = postcode.to_s.strip.presence if postcode
+    self.postcode = postcode.gsub(" ", "").presence if postcode
```

Comment:
I was thinking about it before and I think as we work more on the postcode implementation, it's becoming important. I would normalise the input into a standard format postcode. We can use the `uk_postcode` gem for this (also for validation). 
It's good to store the postcodes in the standard all caps format even if we accept lower case characters. It makes life a lot easier if all postcodes in the database are in the same format, with the space, etc.

Reply:
We can store them as upper case, no problem, it was just the vcr cassettes differentiated between upper/lower case uri's

Reply:
Have upcased, but postcodes.io will not accept TR1 1UF in the uri, it must be TR11UF, so I don't think uk_postcodes will be any use to us?

Reply:
Oh, right. We can strip the space for the api call (in the `PostcodeApi` service). But I'd still store the normalise format in the DB, if possible 🤔

Reply:
Sounds good